jib-core/CHANGELOG.md

```diff
@@ -9,6 +9,8 @@ All notable changes to this project will be documented in this file.
 
 ### Changed
 
+- Upgraded jib-build-plan to 0.3.0. ([#2594](https://github.com/GoogleContainerTools/jib/pull/2594))
```

Comment:
0.3.1

Reply:
Is this supposed to be a `CHANGELOG` for 0.3.0 though?

Reply:
Good question. Let me explain. Note that the top CHANGELOG entry is `## [unreleased]` and the second entry is `## 0.15.0`. That is, the latest jib-core version we released and is available on the Internet (on the repository called Maven Central) is 0.15.0. Now, until we make a new jib-core release (which could be 0.16.0, for example), we write under `## [unreleased]` all important and publicly visible changes that the user needs/wants to know about that future release. The current jib-core 0.15.0 is using jib-build-plan 0.2.0, while the next jib-core release will use jib-build-plan 0.3.1. This is exactly what should be documented and also the truth.

Reply:
Thanks for the explanation @chanseokoh  ,now i understand how the release cycle works better.